src/components/tables/TableDataModifier.ts

```diff
@@ -13,20 +13,20 @@ import { ITableCompositeState, ITableState } from './TableReducers';
 import { unselectAllRows } from './TableRowActions';
 import { getTableChildComponentId, getTableLoadingIds } from './TableUtils';
 
-export const dispatchPreTableStateModification = (tableOwnProps: ITableOwnProps, dispatch: IDispatch) => {
-  dispatch(unselectAllRows(tableOwnProps.id));
+export const dispatchPreTableStateModification = (id: string, dispatch: IDispatch) => {
```

Comment:
can you consider naming `id` for `tableId` here? Since there are a lot of components interrelated/moving together in the table it's good to remain explicit in this context imho.